src/main/java/calculator/Calculation.java

```diff
@@ -0,0 +1,39 @@
+package calculator;
+
+import java.util.Arrays;
+import java.util.function.BiFunction;
+
+public enum Calculation {
+    PLUS("+", (a, b) -> a + b),
+    MINUS("-", (a, b) -> a - b),
+    MULTIPLE("*", (a, b) -> a * b),
+    DIVIDE("/", (a, b) -> {
+        validToDivideByZero(b);
+        return a / b;
+    });
+
+    private String expressionLabel;
+    private BiFunction<Integer, Integer, Integer> calculateValue;
+
+    Calculation(String expressionLabel, BiFunction<Integer, Integer, Integer> calculateValue) {
+        this.expressionLabel = expressionLabel;
+        this.calculateValue = calculateValue;
+    }
+
+    public static int calculateValue(String expressionLabel, int firstValue, int lastValue) {
+        return getExpression(expressionLabel).calculateValue.apply(firstValue, lastValue);
+    }
+
+    private static Calculation getExpression(String expressionLabel) {
+        return Arrays.stream(values())
+                .filter(o -> o.expressionLabel.equals(expressionLabel))
+                .findFirst().orElseThrow(() -> new IllegalArgumentException("올바른 연산자가 아닙니다."));
+    }
```

Comment:
```suggestion
    private static Calculation getExpression(String expressionLabel) {
        return Arrays.stream(values())
                .filter(o -> o.expressionLabel.equals(expressionLabel))
                .findFirst()
                .orElseThrow(() -> new IllegalArgumentException("올바른 연산자가 아닙니다."));
    }
```

스트림을 잘 사용해주셨습니다. 스트림은 항상 순차적으로 연산이 일어나는 것을 기대할 수 있으므로 개행으로 가독성을 높여주시는 것을 추천드립니다.